.eslintrc

```diff
@@ -41,7 +41,12 @@
     ],
     "prettier/prettier": "error",
     "compat/compat": "error",
-    "linebreak-style": "off"
+    "linebreak-style": "off",
+    "jsx-a11y/media-has-caption": [ 1, {
```

Comment:
I thought we managed to fix this by putting a default Track

Reply:
yes, removed and this fixed has been done in other branch 162